conf/documentation.conf

```diff
@@ -1003,7 +1004,7 @@ lower the value.
 EOT
 
 [captive_portal.image_path]
-type=text
+type=path
```

Comment:
The Path form field checks if a path exists on the file system.
Please convert this back to text.